### PR TITLE
(feat) Chart.js 2.0-dev compatibility

### PR DESCRIPTION
**Pull request by danmolitor**
After reading about Chart.js 2.0's performance upgrades, I've updated react-chartjs to be compatible with the Chart.js most up to date 2.0-dev branch, including the new bubble and scatter charts. By moving from 1.0.2 to 2.0, the time it takes a simple line-chart to render has dropped significantly.

Since people may still want to use Chart.js v1.0.2., if you don't want to merge yet people can still use this version by updating their dependencies in their project's package.json to:

"chart.js": "git://github.com/danmolitor/Chart.js.git#v2.0-dev",
"react-chartjs": "git+https://github.com/jhudson8/react-chartjs#chartjs-v2"

I've attached a image showing the original charts using Chart.js v2.0-dev.

react-chartjs2

### CODE REVIEW

**Comment by austinpray** [comment]
I edited your PR comment slightly to reflect "jhudson8/react-chartjs#chartjs-v2"

**Comment by danmolitor** [comment]
Okay, great! Let me know if there's anything else I can do.

File: dist/react-chartjs.js [hunk 1]
```diff
@@ -110,8 +119,10 @@ return /******/ (function(modules) { // webpackBootstrap
 
 	    classData.componentDidMount = function() {
 	      this.initializeChart(this.props);
+	      console.log(this.props);
```
Comment (jamiepenney) [inline dist/react-chartjs.js]: Should probably get rid of this console.log()

Comment (austinpray) [inline dist/react-chartjs.js]: :sunglasses:

Comment (danmolitor) [inline dist/react-chartjs.js]: Oh, forgot about that one! Gone now.

**Comment by austinpray** [comment]
Thank you! Upward and onward!

**Pull request merged**